buildSrc/src/main/groovy/org/elasticsearch/gradle/NoticeTask.groovy

```diff
@@ -40,6 +42,10 @@ public class NoticeTask extends DefaultTask {
     /** Configurations to inspect dependencies*/
     private List<Project> dependencies = new ArrayList<>()
 
+    /** Extra directories to include notices from */
+    @InputDirectory
```

Comment:
Annotations like this won't do anything when the member is private. These helper annotations are meant to add helper methods based on the input type.